src/cmd/policy.go

```diff
@@ -84,6 +91,30 @@ func RegoFuncReadFile(ctx rego.BuiltinContext, a *ast.Term) (*ast.Term, error) {
 	return nil, nil
 }
 
+func RegoFuncGetGithubRepo(ctx rego.BuiltinContext, a, b *ast.Term) (*ast.Term, error) {
+
+	var org, repo string
+
+	if err := ast.As(a.Value, &org); err != nil {
+		return nil, err
+	} else if ast.As(b.Value, &repo); err != nil {
+		return nil, err
+	}
+
+	flags := runCmd.Flags()
+	githubToken, err := flags.GetString("github-token")
+	cobra.CheckErr(err)
+
+	response, err := getClientRest().R().
+		SetHeader("Accept", "application/vnd.github+json").
+		SetHeader("Authorization", fmt.Sprintf("token %v", githubToken)).
+		Get(fmt.Sprintf("https://api.github.com/repos/%v/%v", org, repo))
+
+	cobra.CheckErr(err)
+
+	return ast.StringTerm(response.String()), nil
```

Comment:
This still needs work... need to find a way to return an object instead of a string.